src/java2yaml/Steps/GenerateDocument.cs

```diff
@@ -13,7 +13,22 @@ public Task RunAsync(ConfigModel config)
         {
             return Task.Run(() =>
             {
-                //TODO
+                foreach (var path in config.RepositoryFolders)
+                {
+                    var procedure = new StepCollection(
+                        new RestoreDependency(path),
+                        new RunJavadoc(path));
+
+                    try

```

Comment:
Feels no need to try/catch here. `StepCollection.RunAsync()` already has a try/catch wrapping this.

Reply:
removed unnecessary try/catch